python/Ganga/Core/MonitoringComponent/Local_GangaMC_Service.py

```diff
@@ -664,10 +665,7 @@ def runMonitoring(self, jobs=None, steps=1, timeout=300, _loadCredentials=False)
         if not self.enabled:
             # and there are some required cred which are missing
             # (the monitoring loop does not monitor the credentials so we need to check 'by hand' here)
-            if _loadCredentials is True:
-                _missingCreds = Coordinator.getMissingCredentials()
-            else:
-                _missingCreds = False
```

Comment:
Because the default for `runMonitoring` was `False` and never called with something else (as far as I can tell), it seems the missing credentials was never checked. This is changed in the new code which I think I agree with but just wanted to check this was expected.